_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,68 @@
+- [Reporting Dashboards Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Trace Analytics Dashboards Plugin](https://github.com/opensearch-project/trace-analytics/blob/main/release-notes/opensearch-trace-analytics.release-notes-1.0.0.0-rc1.md)
+- [Gantt Charts Dashboards Visualization Plugin](https://github.com/opensearch-project/dashboards-visualizations/blob/main/release-notes/opensearch-dashboards-visualizations.release-notes-1.0.0.0-rc1.md)
+- [Async-Search](https://github.com/opensearch-project/asynchronous-search/blob/main/release-notes/opensearch-asynchronous-search.release-notes-1.0.0.0-rc1.md)
+- [Common-Utils](https://github.com/opensearch-project/common-utils/blob/main/release-notes/opensearch-common-utils.release-notes-1.0.0.0-rc1.md)
+
+**What does this Release Candidate not include?** 
+
+* *Minimum Artifacts* - Based on community feedback, we plan on releasing a standalone, minimal downloadable artifacts without the OpenSearch plugins (alerting, AD, security, etc…). We are expecting to deliver these artifacts by GA.
+* *Additional Artifacts* - In addition to the current artifacts (Linux tars and Docker images) we are hoping to deliver RPM (X64), Windows (X64), and DEB (X64) by GA and Tar (ARM64), DEB (ARM64), RPM (ARM64), MacOS (M1 ARM), and MacOS (X64) by OpenSearch 1.1. 
+* *Clients with License Checks* - We are aware of the issue with license checks which are causing incompatibilities (i.e. the recent release of Logstash & Beats 7.13) and are working on a solution for the community before GA.
+
+**How can you help?** 
+
+The best thing you can do is run the Release Candidate through its paces. Every bit of testing or feedback the community can provide helps ensure that the changes are doing what they should and we are building to the best end product. Couple specific places to start;
+
+- Install the Release Candidate and open an [issue](https://github.com/opensearch-project/OpenSearch/issues) to report any bugs you discover. 
+- Run rolling upgrades from existing version to OpenSearch RC1 in a sandbox cluster. 
+- Test clients! We want to know what is working (and not working) for you. 
+
+Before installing or upgrading to OpenSearch 1.0.0 or OpenSearch Dashboards 1.0.0, please read the release notes ([OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.0.0-rc1.md) and [OpenSearch Dasboard](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.0.0-rc1.md)). While we consider this 1.0.0 release candidate suitable for testing by the community, you should not use a release candidate in a production environment - this release candidate is provided for testing and validation purposes only.
```

Comment:
I'm not sure where to put it but we should link to the consolidated release notes in the build repo.